integrations/aws/.port/resources/port-app-config.yml

```diff
@@ -0,0 +1,68 @@
+resources:
+  - kind: AWS::Organizations::Account
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .Id
+          title: .Name
+          blueprint: '"awsAccount"'
+          properties:
+            arn: .Arn
+            email: .Email
+            status: .Status
+            joined_method: .JoinedMethod
+            joined_timestamp: .JoinedTimestamp
+
+  - kind: AWS::S3::Bucket
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .Identifier
+          title: .Identifier
+          blueprint: '"cloudResource"'
+          properties:
+            kind: .__Kind
+            region: .__Region
+            tags: .Properties.Tags
+            arn: .Properties.Arn
+            link: '.Properties | select(.Arn != null) | "https://console.aws.amazon.com/go/view?arn=" + .Arn'
+          relations:
+            account: .__AccountId
+  - kind: AWS::EC2::Instance
+    selector:
+      query: 'true'
+    port:
+      entity:
+        mappings:
+          identifier: .Identifier
+          title: .Identifier
```

Comment:
do we have title we can use?

Reply:
unfortunately there's is no property that I can account on that is the same for all the resources (i.e I can only rely on the integration custom properties)